examples/common/eventdisplay/FairEveMCTracks.cxx

```diff
@@ -149,8 +149,15 @@ InitStatus FairEveMCTracks::Init()
 {
     FairRootManager *mngr = FairRootManager::Instance();
     fContainer = (TClonesArray *)mngr->GetObject("MCTrack");
-    if (fContainer == nullptr)
-        return kFATAL;
+    if (fContainer == nullptr) {
+        LOG(warning) << "MCTrack branch not found !";
+        return kERROR;
+    }
+    TClass *classTrack = fContainer->GetClass();
+    if (!classTrack->InheritsFrom("FairMCTrack")) {
```

Comment:
```suggestion
    if (classTrack && !classTrack->InheritsFrom("FairMCTrack")) {
```

Unless you can make a compelling argument, why this cannot return `nullptr` here?